sdk/appconfiguration/app-configuration/test/public/utils/testHelpers.ts

```diff
@@ -42,10 +42,7 @@ export function startRecorder(that: any): Recorder {
       AZURE_CLIENT_SECRET: "azure_client_secret",
       AZURE_TENANT_ID: "azuretenantid"
     },
-    customizationsOnRecordings: [
-      (recording: any): any =>
-        recording.replace(/"access_token":"[^"]*"/g, `"access_token":"access_token"`)
-    ],
+    customizationsOnRecordings: [],
```

Comment:
could we make this optional so don't need to pass empty arrays? Similar for `queryParametersToSkip` below

Reply:
Can be done!